src/components/rating.ts

```diff
@@ -1,11 +1,9 @@
-import {Component, View, NgFor, NgClass, EventEmitter, OnInit} from 'angular2/angular2';
+import {Component, NgFor, NgClass, EventEmitter, OnInit} from 'angular2/angular2';
 
 @Component({
   selector: 'ngb-rating',
   inputs: ['max', 'rate', 'readonly'],
-  outputs: ['updateRating: rate', 'onHover', 'onLeave']
-})
-@View({
+  outputs: ['updateRating: rate', 'onHover', 'onLeave'],
```

Comment:
This is just a decorator that you stick on a class member (property) to mark it as input / output.